options/internal/include/mlibc/internal-sysdeps.hpp

```diff
@@ -35,6 +36,9 @@ int sys_close(int fd);
 int sys_vm_map(void *hint, size_t size, int prot, int flags, int fd, off_t offset, void **window);
 int sys_vm_unmap(void *pointer, size_t size);
 
+int sys_sigentry(void *pointer);
+[[noreturn]] int sys_sigreturn(void *context, sigset_t old_mask);
```

Comment:
Since this is used only on Vinix, it would be better to keep it within the sysdeps.

Reply:
Done